docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc

```diff
@@ -105,6 +105,31 @@ To work around this problem, uninstall the {agent} and install it again with
 {fleet-server} enabled during the bootstrap process.
 ====
 
+--
```

Comment:
This `--` is causing the build to fail. Remove this and you should get a passing build.

```suggestion
```